c2corg_ui/static/partials/cards/images.html

```diff
@@ -0,0 +1,16 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <img ng-src="{{cardCtrl.createImg()}}" alt="{{locale['title']}}">
```

Comment:
OK for testing with MI, I was confused thinking you wanted to use the BI images. MI are not that bigger than SI and preserve the whole image, on the contrary of SI that are cropped squared images.